vm/adb/adb.go

```diff
@@ -338,6 +338,14 @@ func (inst *instance) repair() error {
 	// Switch to root for userdebug builds.
 	inst.adb("root")
 	inst.waitForSSH()
+
+	// ADB connects to a phone and starts syz-fuzzer while the phone is still booting.
```

Comment:
Is there anything more specific to wait for? This just adds 1 minute to every boot. Is either wastes too much time, or will lead to flakes.
Maybe there is some service that needs to come up? E.g. in getBatteryLevel we wait for some battery service.